# This file was autogenerated by uv via the following command:
#    nox -s update_python_dependencies
cmake==3.31.4 \
    --hash=sha256:20be7cdb41903edf85e8a498c4beff8d6854acbb087abfb07c362c738bdf0018 \
    --hash=sha256:225d9a643b0b60ffce0399ff0cabd7a4820e0dbcb794e97d3aacfcf7c0589ae6 \
    --hash=sha256:23781e17563693a68b0cef85749746894b8a61488e56e96fc6649b73652e8236 \
    --hash=sha256:25c5094394f0cee21130b5678e5b4552f72470e266df6d6fb1d5c505100f0eaa \
    --hash=sha256:2a37be93534df04513f0845492d71bc80899c3f87b77e3b01c95aff1a7fc9bde \
    --hash=sha256:466c9295af440bb4a47cc5e1af10576cf2227620528afd0fd0b3effa1d513b49 \
    --hash=sha256:838a388b559137f3654d8cf30f62bbdec10f8d1c3624f0d289614d33cdf4fba1 \
    --hash=sha256:89143a5e2a5916061f2cfc5012e9fe6281aaf7c0dae7930bdc68d105d22ddc39 \
    --hash=sha256:926d91cae2ba7d2f3df857d0fc066bdac4f3904bf5c95e99b60435e85aabedb4 \
    --hash=sha256:929a8d8d289d69e43784661748ddd08933ce1ec5db8f9bcfce6ee817a48f8787 \
    --hash=sha256:9479a9255197c49e135df039d8484c69aa63158a06ae9c2d0eb939da2f0f7dff \
    --hash=sha256:a6a3b0b9557f41c955a6b25c94205f2ca9c3a46edca809ad87507c5ef6bc4274 \
    --hash=sha256:a6ac2242e0b16ad7d94c9f8572d6f232e6169747be50e5cdf497f206c4819ce1 \
    --hash=sha256:b463efdf5b92f3b290235aa9f8da092b3dac19b7636c563fd156022dab580649 \
    --hash=sha256:c9f5f8289c5e7bd2ed654cbac164021fa7723064fee0443a2f0068bc08413d81 \
    --hash=sha256:d378c9e58eac906bddafd673c7571262dcd5a9946bb1e8f9e3902572a8fa95ca \
    --hash=sha256:f6af3b83a1b1fc1d990d18b6a566ee9c95c0393f986c6df15f2505dda8ad1bcc \
    --hash=sha256:f96127bf663168accd29d5a50ee68ea80f26bcd37f96c7a14ef2378781f19936 \
    --hash=sha256:fc048b4b70facd16699a43c737f6782b4eff56e8e6093090db5979532d9db0f6
    # via -r .nox/update_python_dependencies/tmp/cmake.in
